webpack.config.js

```diff
@@ -35,7 +35,12 @@ module.exports = function (env = {}) {
         {
           test: /\.js$/,
           exclude: /(node_modules|bower_components)/,
-          use: "babel-loader",
+          use: [{
+            loader: "babel-loader",
+            options: {
+              plugins: ["syntax-dynamic-import"],
```

Comment:
Can you put this into the `.babelrc`? Otherwise, executingn babel without webpack won't work.